sopel/modules/search.py

```diff
@@ -12,9 +12,10 @@
 
 if sys.version_info.major < 3:
     from urllib import quote_plus
+    from urllib import unquote
 else:
     from urllib.parse import quote_plus
-
+    from urllib.parse import unquote
```

Comment:
Can you collapse these imports like `from urllib.parse import quote_plus, unquote`?

And flake8 will complain about removing the second blank line before `def formatnumber`.